src/gtk-4.0/widgets/adw/PreferencesWindow.scss

```diff
@@ -0,0 +1,50 @@
+window.preferences.csd {
+    @extend .dialog;
+
+    headerbar {
+        @extend .flat;
+
+        viewswitcher button.toggle {
+            border-radius: rem(3px);
+
+            image {
+                opacity: 0;
+                -gtk-icon-size: 0;
+            }
+        }
+    }
+
+    preferencespage > scrolledwindow {
+        undershoot.top {
+            background:
+                linear-gradient(
+                    $toplevel-border-color,
+                    rgba(black, 0.05) 1px,
+                    rgba(black, 0.0) 0.444rem
+                );
+        }
+
+        > viewport > clamp > box {
+            padding-left: rem(12px);
+            padding-right: rem(12px);
+        }
+    }
+
+    preferencesgroup {
+        margin-bottom: 2rem;
+    }
+
+    leaflet stack + actionbar {
```

Comment:
Pref window doesn't have an actionbar, does it? Do you mean view switcher bar, or action bars everywhere?

Reply:
This targets the actionbar that appears on the passwords page

Reply:
1. That's an ephy-specific thing. Why style it here and not in ephy?
2. That's just a regular actionbar, what does it have to do with stack/leaflet/prefwindow?

Reply:
That's a good point. I should probably add a style sheet to our Ephy build for those kinds of specific tweaks

It was mainly just to try to make sure I'm selecting that specific actionbar instead of the view switcher one

Reply:
You know we have one upstream too, right? ;)

Reply:
Also that action bar is also there in the history window.